packages/walt-compiler/src/tokenizer/comments/index.js

```diff
@@ -2,17 +2,53 @@
 import token from "../token";
 import Syntax from "../../Syntax";
 
-const everything = () => everything;
+import {
+  SLASH,
+  SINGLE_LINE,
+  MULTI_LINE_START,
+  MULTI_LINE_END,
+  COMMENT_IDENTIFIERS,
+  everything,
+} from "./constants";
 
-const slash = char => {
-  if (char === "/") {
-    return everything;
+let isMultiline: boolean = false;
+let previous: string;
+
+const isComment = (current: string) => {
+  let value = null;
```

Comment:
I don't think we need to create this variable at all if we just return from all of the `case`s below.

Reply:
That's exactly how i'd done it initially but for some reason i reverted to this approach.